input/intro-notes/StructureDefinition-au-relatedperson-intro.md

```diff
@@ -2,3 +2,5 @@
 
 **Profile specific implementation guidance:**
 - See each Identifier profile page for guidance related to that identifier type.
+- When exchanging concepts of sex or gender, refer to the guidance in the [Gender Harmony Implementation Guide](http://hl7.org/xprod/ig/uv/gender-harmony/).
```

Comment:
Missing the implementation guidance that was present. Put back but change it to remove the reference to the examples.

Reply:
I see it reinstated but have lost the following point that needs to be present:
- When exchanging concepts of sex or gender, refer to the guidance in the [Gender Harmony Implementation Guide](http://hl7.org/xprod/ig/uv/gender-harmony/).